client/src/components/Settings/EmailForm.js

```diff
@@ -9,6 +9,7 @@ import { getSettings, updateSettings } from '../../actions/settingsActions.js';
 import { withStyles } from '@material-ui/core/styles';
 import TextField from '@material-ui/core/TextField'; 
 import Button from '@material-ui/core/Button';
+import Typography from "@material-ui/core/Typography";
```

Comment:
For all material-ui/core you can add them in one line to reduced all the imports. Might make it look a little cleaner.